Sming/Core/Network/Http/HttpRequest.cpp

```diff
@@ -83,20 +83,20 @@ void HttpRequest::reset()
 	files.clear();
 }
 
-#ifndef SMING_RELEASE
 String HttpRequest::toString()
 {
 	String content;
-	content += String(http_method_str(method)) + ' ' + uri.getPathWithQuery() + _F(" HTTP/1.1\n");
+	content += String(http_method_str(method)) + ' ' + uri.getPathWithQuery() + _F(" HTTP/1.1\r\n");
 	content += headers.toString(HTTP_HEADER_HOST, uri.getHostWithPort());
 	for(unsigned i = 0; i < headers.count(); i++) {
 		content += headers[i];
 	}
 
 	if(bodyStream != nullptr && bodyStream->available() >= 0) {
 		content += headers.toString(HTTP_HEADER_CONTENT_LENGTH, String(bodyStream->available()));
+	} else {
+		content += _F("\r\n");
```

Comment:
Don't bother with _F, "\r\n" is very commonly used so linker will merge all instances.